fars/booking/templates/base.html

```diff
@@ -43,17 +43,12 @@
 
       <!-- If not logged in, show form for logging in -->
       {% else %}
-        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapseContent" aria-controls="collapseContent" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
navbar-toggler seems to work just fine for me when testing locally (tested on recent chrome & firefox versions)

Reply:
Yeah, seems like I entirely misunderstood what that code tried to do. Definitely should not be removed, thanks for pointing it out. Reverting that commit!